taxcalc/tests/test_dropq.py

```diff
@@ -33,10 +33,24 @@
   "behavior": {
     "_BE_sub": {"2016": [0.25]}
   },
+  "consumption": {
```

Comment:
Is it correct that REFORM_CONTENTS should have behavior, consumption, and growth? Or should that be only in ASSUMP_CONTENTS?